quick-start.md

```diff
@@ -58,13 +58,42 @@ Send the policy to Vault: `vault policy-write kubernetes-vault policy.hcl`
 Create a token role for Kubernetes-Vault that generates a 6 hour periodic token: `vault write auth/token/roles/kubernetes-vault allowed_policies=kubernetes-vault period=6h`
 
 ### 2.6 Generate the token for Kubernetes-Vault and AppID
-Generate the token: `vault token-create -role=kubernetes-vault`
+Generate the token: `vault token-create -role=kubernetes-vault`. And make a note of the token output. In the example below it would be '00000000-1111-2222-3333-444444444444'
+
+```
+$ vault token-create -role=kubernetes-vault
+
+Handling connection for 8200
+Key             Value
```

Comment:
I think the `Handling connection for 8200` line can be removed as it's extraneous. If you run `kubectl port-forward vault-xxxxxxxx 8200` in a separate terminal, it should not be appear in the terminal you are running `vault` commands from.